src/sapling/saplingscriptpubkeyman.cpp

```diff
@@ -374,6 +374,37 @@ std::vector<libzcash::SaplingPaymentAddress> SaplingScriptPubKeyMan::FindMySapli
     return ret;
 }
 
+void SaplingScriptPubKeyMan::GetNotes(const std::vector<SaplingOutPoint>& saplingOutpoints,
+                                      std::vector<SaplingNoteEntry>& saplingEntriesRet)
```

Comment:
nit: in the future this could be refactored to remove the code duplication, adding a per-outpoint filter to  `SaplingScriptPubKeyMan::GetFilteredNotes`

Reply:
yeah, nice.